assets/js/blocks/product-results-count/edit.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { useBlockProps } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+export interface Attributes {
+	className?: string;
+}
+
+const Edit = () => {
+	const blockProps = useBlockProps( {
+		className: 'wc-block-product-results-count',
+	} );
+
+	return (
+		<>
+			<div { ...blockProps }>
+				<div>
+					{ __(
+						'Showing 1-X of X results',
+						'woo-gutenberg-products-block'
+					) }
+				</div>
+			</div>
+		</>
+	);
+};
```

Comment:
Nitpick, but I think we should match the markup between frontend and editor.

```suggestion
const Edit = () => {
	const blockProps = useBlockProps( {
		className: 'woocommerce wc-block-product-results-count',
	} );

	return (
		<div { ...blockProps }>
			<p className="woocommerce-result-count">
				{ __(
					'Showing 1-X of X results',
					'woo-gutenberg-products-block'
				) }
			</p>
		</div>
	);
};
```